KsApi/fragments/ProjectFragment.graphql

```diff
@@ -17,6 +22,13 @@ fragment ProjectFragment on Project {
   description
   finalCollectionDate
   fxRate
+  friends {
+    edges {
```

Comment:
I typically only use edges if I need information from the edge, in this case we could probably just do:
```graphql
friends {
  nodes {
    ...UserFragment
  }
}
```
But same result really.